### DISCUSSION TITLE
taking first step

### DISCUSSION THREAD

**Discussion by anshdev-dev** [Cyfrin/foundry-full-course-cu#2343] [General]
hi everyone my name is Devansh. I am a college pass out and I am starting my blockchain developer journey from this course   and I need some big brothers, sister ,mentors who guide me on this path so please help me

**Comment by EngrPips** [comment]
Welcome aboard @anshdev-dev, We are here to help and support you.

**Reply by anshdev-dev** [reply]
thank you

**Comment by AboloreDev** [comment]
This community is here to support you all the way. Welcome @anshdev-dev .

**Reply by EngrPips** [reply]
Indeed

**Reply by anshdev-dev** [reply]
thank you

**Comment by cmed00** [comment]
We're here with you Bro. if you don't understand something, Don't hesitate to ask questions !

**Reply by anshdev-dev** [reply]
thank you

**Comment by dannyakorede12** [comment]
You are Welcome to the community Devansh

**Reply by anshdev-dev** [reply]
thank you

**Comment by fighting45** [comment]
Hey @anshdev-dev welcome aboard friend. We all are here to learn :) 
lets connect https://www.linkedin.com/in/usama-hassan-8081401b8/

**Comment by suvasis-patra** [comment]
Hello everyone! I am Suvasis, currently a final-year student at IIT KGP. I am starting my journey in blockchain development with this course.  Want your valuable advice to complete this course.

**Reply by EngrPips** [reply]
Code along with Patrick and comment on every single line of code you write, explaining what that line of code does.
